h2/src/main/org/h2/util/MemoryEstimator.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2004-2020 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (https://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.util;
+
+import static org.h2.engine.Constants.MEMORY_POINTER;
+import org.h2.mvstore.type.DataType;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Class MemoryEstimator.
+ * <UL>
+ * <LI> 12/7/19 10:45 PM initial creation
+ * </UL>
+ *
+ * @author <a href='mailto:andrei.tokar@gmail.com'>Andrei Tokar</a>
```

Comment:
would be nice to copy/paste the PR description you provided as class javadoc here